dataactvalidator/config/detachedAwardFields.csv

```diff
@@ -5,7 +5,7 @@ assistancetype,assistance_type,TRUE,str,2,D1,TRUE
 awarddescription,award_description,FALSE,str,4000,D2,
 awardeeorrecipientlegalentityname,awardee_or_recipient_legal,TRUE,str,120,D22,
 awardeeorrecipientuniqueidentifier,awardee_or_recipient_uniqu,FALSE,str,9,"D2, D10",
-awardingagencycode,awarding_agency_code,FALSE,str,3,D1,
+awardingagencycode,awarding_agency_code,TRUE,str,3,D1,
```

Comment:
We probably want to add padding to this so it pads out to 3 digits instead of auto-cutting if someone enters, say "020"